src/applications/financial-status-report/containers/ConfirmationPage.jsx

```diff
@@ -45,7 +45,7 @@ const RequestDetailsCard = ({ data, response }) => {
     );
   };
 
-  const reliefList = combinedFSR
+  const reliefList = enhancedFSR
```

Comment:
I think we can remove this flag entirely because the ternary used the `selectedDebts`.